xchange-stream-bitmex/src/main/java/info/bitrich/xchangestream/bitmex/dto/BitmexWebSocketTransaction.java

```diff
@@ -90,6 +92,15 @@ public BitmexFunding toBitmexFunding() {
     return funding;
   }
 
+  public RawOrderBook toRawOrderBook() {
+    try {
+      return mapper.treeToValue(this.data.get(0), RawOrderBook.class);
+    } catch (JsonProcessingException e) {
+      e.printStackTrace();
```

Comment:
would prefer to log using logging framework, log line will be explicit with location and what is triggering this error no need for stack trace

Reply:
fixed in all exceptions handling